beta-component-promotion/README.md

```diff
@@ -23,21 +23,16 @@ as well as relevant examples in the documentation are all labeled as beta.
 
 | *Component currently in beta*          | inital @patternfly/patternfly milestone | inital @patternfly/patternfly-react milestone |
 |----------------------------------------|-----------------------------------------|-----------------------------------------------|
-| Button: with count                     | 2022.11                                 | 2022.12                                       |
-| DataList: draggable                    | 2021.12                                 | 2020.13                                       |
-| Drag and drop                          | 2021.13                                 | 2021.14                                       |
-| Date and time picker demo              | --                                      | 2020.16                                       |
-| Expandable section: truncate expansion | 2022.11                                 | 2022.11                                       |
-| Label: Editable                        | 2021.07                                 | 2021.10                                       |
-| LabelGroup: Editable                   | 2022.08                                 | 2022.08                                       |
-| Menu: flyout                           | 2020.11                                 | 2021.10                                       |
-| Menu: drilldown                        | 2020.11                                 | 2021.03                                       |
-| Nav: flyout                            | 2021.13                                 | 2021.13                                       |
-| Nav: drilldown                         | 2021.12                                 | --                                            |
-| Pagination: insets                     | 2022.15                                 | 2022.16                                       |
-| Spinner: inline                        | 2022.14                                 | 2022.15                                       |
+| Chart skeleton theme                   | --                                      | 2024 Q2                                       |
 | Table: draggable rows                  | 2021.08                                 | 2021.08                                       |
```

Comment:
Should we update the table draggable rows examples in core to show the beta badge? https://github.com/patternfly/patternfly/blob/aa31c48cb562da64a2615365141201e021eb1657/src/patternfly/components/Table/examples/Table.md?plain=1#L3769

Reply:
We could. They've always been marked beta in react.